src/main/java/org/folio/oaipmh/processors/MarcWithHoldingsRequestHelper.java

```diff
@@ -160,6 +160,49 @@ public static MarcWithHoldingsRequestHelper getInstance() {
     return promise.future();
   }
 
+  private void updateLastUpdateDateForRequestMetadata(String requestId, PostgresClient postgresClient, String tenantId) {
```

Comment:
Need to change with additional jooq dao method which will not take more then 10 lines of code at maximum